Gopkg.toml

```diff
@@ -11,7 +11,8 @@ required = [
 # TODO switch to tags before PMM 2.0 release
 [[constraint]]
   name = "github.com/percona/pmm"
-  branch = "PMM-2.0"
+  # branch = "PMM-2.0"
+  branch = "PMM-4226-sparkline-for-main-metric-only"
```

Comment:
https://github.com/percona/pmm/pull/391 should be finished first.

Reply:
Yeap, resolved conflicts.